docs/user_guide/releases.rst

```diff
@@ -71,15 +71,20 @@ Release branches
      - Vivado 2023.2
      -
      -
+   * - :git-hdl:`hdl_2023_r2 <hdl_2023_r2:>`
+     - Quartus Pro 23.2
+     - Vivado 2023.2
+     - `Release notes 2023_R2 <https://github.com/analogdevicesinc/hdl/releases/tag/2023_R2>`_
+     - `Wiki list for hdl_2023_r2 <https://wiki.analog.com/resources/fpga/docs/hdl/downloads_2023_r2>`_
    * - :git-hdl:`hdl_2022_r2 patch1 <2022_r2_p1:>`
      - Quartus Pro 22.4
      - Vivado 2022.2
-     - `Release notes 2022_R2 <https://github.com/analogdevicesinc/hdl/releases/tag/2022_r2_p1>`
+     - `Release notes 2022_R2 <https://github.com/analogdevicesinc/hdl/releases/tag/2022_r2_p1>`_
```

Comment:
Because otherwise this generates a "WARNING: Duplicate explicit target name: "release notes 2022_r2".
```suggestion
     - `Release notes 2022_R2_p1 <https://github.com/analogdevicesinc/hdl/releases/tag/2022_r2_p1>`_
```

Reply:
I agree with appending the `_p1` since it is the full release name.
Still, to avoid the warning, use a anonymous target name by using double underlines instead of single underline at the end (for things like ``` `read here <some_link>`_ ```).

Reply:
> I agree with appending the `_p1` since it is the full release name. Still, to avoid the warning, use a anonymous target name by using double underlines instead of single underline at the end (for things like `` `read here <some_link>`__ ``).

It worked, thank you.